pytest_fixtures/component/maintain.py

```diff
@@ -0,0 +1,10 @@
+# Satellite-maintain fixtures
+import pytest
+
+from robottelo.config import settings
+from robottelo.hosts import Satellite
+
+
+@pytest.fixture(scope='module')
+def remotedb_sat():
```

Comment:
```suggestion
def module_remotedb_sat():
```

Reply:
As per discussion with @Gauravtalreja1 and since this is more of a sat(broker) fixture than entity fixture, keeping it as it is !